source/engine/graphics/system/SkeletalAnimationTypes.cpp

```diff
@@ -18,6 +18,66 @@ auto GetGlobalInverseTransform(const std::vector<nc::asset::BoneSpaceToParentSpa
 
 namespace nc::graphics::anim
 {
+void State::AddSuccessor(uint32_t successor)
+{
+    successors.push_back(successor);
+}
+
+StableSet::StableSet(uint32_t maxStates)
+    : m_states{},
+      m_sparse(maxStates, NullState)
+{
+    m_states.reserve(maxStates);
```

Comment:
I think we discussed - reserving 100 each time is overkill. Leaving as default initialized is probably fine, but if you want to reserve, I'd do some reasonable average of like 5 (or whatever you think is appropriate).

Reply:
Yup - clobbered when I copy-pasted my working changes with the Code Review's ones afer already making the fix